pr_toolbox/quantum/results/frequencies.py

```diff
@@ -53,13 +63,11 @@ def map_counts(counts: Counts, mapper: Callable) -> Counts:
     Returns:
         New counts with readout bits mapped according to input callable.
     """
-    counts_dict: dict[int, int] = defaultdict(lambda: 0)
-    for readout, freq in counts.int_outcomes().items():
-        readout = mapper(readout)
-        counts_dict[readout] += freq
+    counts_dict: dict[int, int] = map_frequencies(counts.int_outcomes(), mapper)
     return Counts(counts_dict)
 
 
+@map_frequencies.register(QuasiDistribution)
 def map_quasi_dists(quasi_dists: QuasiDistribution, mapper: Callable) -> QuasiDistribution:
```

Comment:
There is only one distribution, so it should be singular (the word distribution already encompasses several frequencies):
```suggestion
def map_quasi_dist(quasi_dist: QuasiDistribution, mapper: Callable) -> QuasiDistribution:
```